### PR TITLE
Reformating zero_pad

### PR DESCRIPTION
**Pull request by ioannispol**

### CODE REVIEW

**Comment by nassimberrada** [comment]
Hi @ioannispol, thank you for your contribution ! The examples are great and you're in the right direction, but i've left some comments above. Could you give those a look as well as correct some lint errors as shown in the `lint / check-formatting` test ? Also, i noticed your PR makes some changes to some files in the .idea folder, namely ivy.iml and misc.xml, could you revert those as they are not needed ? Thanks !

**Review by ioannispol — COMMENTED** [review]
Hi @nassimberrada, thank you for your review. I've corrected the lint errors and revered the changes in the .idea folder (ivy.iml and misc.xml files). Please let me know if I need to make any further changes. 
Thank you!

File: ivy/array/manipulation.py [hunk 1]
```diff
@@ -697,5 +697,37 @@ def zero_pad(
...
+            ((before_1, after_1), … (before_N, after_N)),
+            where N is number of axes of x.
+        out
+            optional output array, for writing the result to.
+            It must have a shape that the inputs broadcast to.
+
+        Returns
+        -------
+        ret
+        Padded array of rank equal to x with shape increased according to pad_width.
```
Comment (nassimberrada) [inline ivy/array/manipulation.py:715]: Could you add an indent just like for other variables above ?

**Review by nassimberrada — CHANGES_REQUESTED** [review]

**Comment by nassimberrada** [comment]
Hi @ioannispol thanks for fixing the lint errors, i think the changes to the ivy.iml files persist though. I made some comments yesterday for changes in the docs but didn't validate them somehow apologies, could you address those ? Thanks !

**Comment by ioannispol** [comment]
Hi @nassimberrada, thanks again for your comments. This time I think I have done them all and tried not to include in my commit the ivy.iml and misc.xml files (I think that PyCharm automatically changes these files, and I mistakenly I committed all the changed files). Please let me know if everything is ok.
Thank you!

File: ivy/container/manipulation.py [hunk 2]
```diff
@@ -1707,6 +1747,47 @@ def zero_pad(
...
+        -------
+        ret
+            Padded array of rank equal to x with shape increased according to pad_width.
+
+        Examples
+        --------
+        With :class:`ivy.Container` input:
+
+        >>> x = ivy.Container(a = ivy.array([1., 2., 3.]), b = ivy.array([3., 4., 5.]))
+        >>> y = ivy.zero_pad(x, pad_width = [[2, 3]])
```
Comment (nassimberrada) [inline ivy/container/manipulation.py:1784]: This should be written like the example in array instance method, i.e x.zero_pad(pad_width=....)

**Review by nassimberrada — CHANGES_REQUESTED** [review]

**Comment by nassimberrada** [comment]
Hi @ioannispol, i think this is almost done, could you just make the last requested change in the container instance method example ? Also, i think your commit still makes changes to the ivy.iml file, could you double check that ? Thanks !

**Comment by ioannispol** [comment]
Hi @nassimberrada, I've corrected the zero_pad example in the container/manipulation.py file. I also doubled check to not commit the files in the .idea folder and it appears that the only file in my fork is the container/manipulation.py. 
Thank you!

**Comment by nassimberrada** [comment]
@ioannispol there were still some changes being made to the ivy.iml but a member of our team rectified that, you may want to keep in mind these unintentional changed in future PRs eventually. In any case, this looks good to me now, i'll be merging, thank you for your contribution !

**Comment by ioannispol** [comment]
Hi @nassimberrada, thank you again for your comments and your help. I'll keep in mind for similar unintentional changes in files such as the ivy.iml. 
Thank you!

**Pull request merged**